README.md

```diff
@@ -8,6 +8,9 @@ DRIVER - Data for Road Incident Visualization, Evaluation, and Reporting
 Requires Vagrant 1.5+, Ansible 1.8+ and the following plugins:
   - `vagrant-hostmanager`
 
+Copy `deployment/ansible/group_vars/all.example` to `deployment/ansible/group_vars/all`
+and add the API key for pickpoint under `web_js_nominatim_key`.
```

Comment:
Maybe add a link to pickpoint. I wasn't familiar with the service before seeing it here.